test/test_habitat_task.py

```diff
@@ -53,6 +54,7 @@ def test_task_actions_sampling_for_teleport():
     config.freeze()
 
     env = habitat.Env(config=config)
+    assert env.action_space.contains(env.action_space.sample())
     env.reset()
     while not env.episode_over:
         action = sample_non_stop_action(env.action_space)
```

Comment:
That maybe even better to move line 57 here:
 `assert env.action_space.contains(action)`